docs/pages/components/upload/examples/ExRounded.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <section>
+    <b-field class="file">
+      <b-upload v-model="file" rounded>
+        <a class="button is-primary is-rounded">
+          <b-icon icon="upload"></b-icon>
+          <span>{{ file.name || "Click to upload"}}</span>
+        </a>
+      </b-upload>
+    </b-field>
```

Comment:
You dont need to use a rounded button if you use this in your example:

```html
<b-field class="file is-primary" :class="{'has-name': !!file}">
        <b-upload v-model="file" class="file-label" rounded>
            <span class="file-cta">
                <b-icon class="file-icon" icon="upload"></b-icon>
                <span class="file-label">{{ file ? file.name : "Click to upload" }}</span>
            </span>
        </b-upload>
    </b-field>
```

Reply:
It would be good to also maange the case we want to use rounded with the more traditional approach and input `type="file"`. Ex.
```html
<b-field class="file is-primary" :class="{'has-name': !!file}">
        <b-upload v-model="file" class="file-label" rounded>
            <span class="file-cta">
                <b-icon class="file-icon" icon="upload"></b-icon>
                <span class="file-label">Click to upload</span>
            </span>
            <span class="file-name" v-if="file">
                {{ file.name }}
            </span>
        </b-upload>
    </b-field>
```
This is the actual behavior:
![image](https://user-images.githubusercontent.com/12817388/97616519-09f91c00-19f3-11eb-99e6-4909f4dcad75.png)

This is the expected behavior:
![image](https://user-images.githubusercontent.com/12817388/97616630-2eed8f00-19f3-11eb-8099-8057ad204165.png)

Reply:
Hey thanks for the review.
It's looking now like this:
![grafik](https://user-images.githubusercontent.com/17907799/98091273-7a85ba00-1e85-11eb-96c2-07e768536d1a.png)

I'm not exactly sure what you mean by the `input type="file"` suggestion. I thought this tag is always created automatically and not by the user?

Reply:
Sorry for the misleading comment about the input. I was talking about displaying the file name like you did in a separate element. In the example, you should set `file: null`. This way, `v-if="file"` in your second example will display the right part only when there is a selected file.

Reply:
All right, now i got what you mean :smile: Thanks for your help :+1: 
I altered the examples so that the second upload button looks now like the first one, until the user actually selected a file.
I think this will be good to go now